src/pages/version.php.ts

```diff
@@ -9,6 +9,11 @@ export const get: APIRoute = async ({ request }) => {
   const version = new URL(request.url).searchParams.get("v");
   if (!version) {
     remote_version = "no_version";
+    // return 400 error if no version is given
+    return new Response("No version given", {
```

Comment:
I don't really care if no version was given, should just return the same thing anyway. 

```suggestion
    return new Response(mqc_releases.latest, {
```